src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
```

Comment:
'manageGroups' method is not ok. I would create two separate methods to handle user's groups:
'addGroup' to add a new group to the user.
prisma object would be like
data: {
groups: {
create: {
group_id: data
}
}
}

'deleteGroup' to remove an existing group from the user
data: {
groups: {
delete: {
group_id: data
}
}
}